xarray/core/dataarray.py

```diff
@@ -3270,9 +3270,8 @@ def map_blocks(
 
             This function cannot add a new chunked dimension.
         args: Sequence
-            Passed verbatim to func after unpacking, after the sliced DataArray. xarray
-            objects, if any, will not be split by chunks. Passing dask collections is
-            not allowed.
+            Passed verbatim to func after unpacking, after the sliced obj.
```

Comment:
What is meant here by the `after the sliced obj`? This seems like an incomplete thought.